Templates/Integrations/CaseManagement/CaseManagement.py

```diff
@@ -0,0 +1,414 @@
+from CommonServerPython import *
+
+''' IMPORTS '''
+from typing import Dict, Tuple, List, AnyStr, Optional, Union
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+"""GLOBALS/PARAMS
+Attributes:
+    INTEGRATION_NAME:
+        Name of the integration as shown in the integration UI, for example: Microsoft Graph User.
+
+    INTEGRATION_COMMAND_NAME:
+        Command names should be written in all lower-case letters,
+        and each word separated with a hyphen, for example: msgraph-user.
+
+    INTEGRATION_CONTEXT_NAME:
+        Context output names should be written in camel case, for example: MSGraphUser.
+"""
+INTEGRATION_NAME = 'Case Management Integration'
+INTEGRATION_NAME_COMMAND = 'case-management'
+INTEGRATION_CONTEXT_NAME = 'CaseManagement'
+
+TIME_FORMAT = '%Y-%m-%dT%H:%M:%S'
+DEFAULT_FETCH_TIME = '3 days'
+
+
+def build_raw_tickets_to_context(tickets: Union[dict, list]):
+    if isinstance(tickets, list):
+        return [build_raw_tickets_to_context(ticket) for ticket in tickets]
+    return {
+        'ID': tickets.get('id'),
+        'Name': tickets.get('name'),
+        'Category': tickets.get('category'),
+        'Description': tickets.get('description'),
+        'Timestamp': tickets.get('timestamp'),
+        'IsOpen': tickets.get('isOpen'),
+        'Assignee': [
+            {
+                'ID': assignee.get('id'),
+                'Name': assignee.get('name')
+            } for assignee in tickets.get('assignee', [])
+        ]
+    }
+
+
+class Client(BaseClient):
+
+    def __init__(self, base_url, limit=50, *args, **kwargs):
+        self._limit = limit
+        super().__init__(base_url, *args, **kwargs)
+
+    def test_module(self) -> Dict:
+        """Performs basic get request to get item samples
+
+        Returns:
+            True if request succeeded
+        """
+        return self._http_request('GET', 'version')
+
+    def list_tickets(self, ticket_id: Optional[AnyStr] = None,
+                     limit: Optional[AnyStr] = None, from_time: Optional[datetime] = None
+                     ) -> dict:
+        """Gets all credentials from API.
+
+        Returns:
+            credentials
+        """
+        suffix = 'ticket'
+        params = dict()
+        if limit:
+            params['limit'] = limit
+        elif self._limit:
+            params['limit'] = limit  # type: ignore
```

Comment:
add error code to `type: ignore` as described here https://github.com/demisto/content/pull/4665